gui/services/models.py

```diff
@@ -2212,3 +2212,80 @@ class ServiceMonitor(Model):
         verbose_name=_("Enable"),
         default=False
     )
+
+
+class NetDataGlobalSettings(Model):
+
+    history = models.IntegerField(
+        default=86400,
+        null=False,
+        blank=False
+    )
+
+    memory_mode = models.CharField(
+        default='save',
+        blank=False,
+        null=False,
+        max_length=10
+    )
+
+    update_every = models.IntegerField(
+        default=1,
+        null=False,
+        blank=False
+    )
+
+    http_port_listen_backlog = models.IntegerField(
+        default=100,
+        null=False,
+        blank=False
+    )
+
+    bind_to = ListField(
+        default=['*'],
```

Comment:
There isnt anywhere else we use `*`, and I dont think ti should be exposed that way.

We should allow only IPs. Empty or 0.0.0.0 means to bind all.